### ISSUE TITLE
Cant enter Room selection with 1.3.0

### ISSUE THREAD

**Issue by dickhaut** [andruzzzhka/BeatSaberMultiplayer#135]
Meta:
Beat Saber Version 1.3.0

With the new version I cant click on "ROOM" after "ONLINE". Nothing happens.
But the "RADIO" works perfectly.

Error Log:

`[CRITICAL @ 08:59:42 | UnityEngine] MissingMethodException: single VRControllersInputManager.TriggerValue(UnityEngine.XR.XRNode)`

**Comment by draven22** [comment]
Yep +1, also seeing this exact issue.

**Comment by Sverto** [comment]
Seems avatarData and avatarHash are null causing this (line 73 and line 222 in PlayerInfo class). Workaround is to check for null value but didn't had the time look into it further (git clone 04/09)

**Comment by dickhaut** [comment]
I dont think that the solution is a `if ... then`. We need to wait until `customAvatar` is ported to 1.3.0.

**Comment by dennismettler** [comment]
same here. i can't press the button "rooms".

**Comment by dennismettler** [comment]
i got the following log:

`[INFO @ 20:55:11 | Beat Saber Multiplayer] Attempting to load JSON @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[ERROR @ 20:55:11 | Beat Saber Multiplayer] Unable to load config @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Creating new config @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Writing to File @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Writing to File @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Added 8 new ServerHubs to config!
[INFO @ 20:55:11 | Beat Saber Multiplayer] Writing to File @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Found 0 presets in RoomPresets folder
[INFO @ 20:55:11 | Beat Saber Multiplayer] Downloading scrapped data...
[INFO @ 20:55:14 | Beat Saber Multiplayer] Received response from github.com...
[INFO @ 20:55:14 | Beat Saber Multiplayer] Parsing scrapped data...
[INFO @ 20:55:14 | Beat Saber Multiplayer] Scrapped data parsed! Time: 0.58s
[INFO @ 20:55:32 | Beat Saber Multiplayer] Hashing all avatars...
[INFO @ 20:55:32 | Beat Saber Multiplayer] All avatars hashed!
[INFO @ 20:55:32 | Beat Saber Multiplayer] Checking for updates...
[INFO @ 20:55:32 | Beat Saber Multiplayer] Loading all avatars...
[INFO @ 20:55:32 | Beat Saber Multiplayer] Loading text shader asset bundle...
[INFO @ 20:55:33 | Beat Saber Multiplayer] Loaded avatar "No Avatar"!
[INFO @ 20:55:33 | Beat Saber Multiplayer] Loaded avatar "Template"!
[INFO @ 20:55:33 | Beat Saber Multiplayer] Loaded avatar "Ooka Miko"!
[INFO @ 20:55:41 | Beat Saber Multiplayer] Requesting channels list...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Requesting channel infos...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Creating message...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Connecting to hub.assistant.moe:3700...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Creating message...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Connecting to hub.auros.red:3700...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Requested channel infos!
[WARNING @ 20:55:42 | Beat Saber Multiplayer] Channel exception: System.Exception: Channel refused connection!
[WARNING @ 20:55:48 | Beat Saber Multiplayer] Channel exception: System.Exception: Channel refused connection!`

**Comment by dennismettler** [comment]
i got the following errorlog: (Firewall disabled and no anti-virus installed)

```[INFO @ 20:55:11 | Beat Saber Multiplayer] Attempting to load JSON @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[ERROR @ 20:55:11 | Beat Saber Multiplayer] Unable to load config @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Creating new config @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Writing to File @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Writing to File @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Added 8 new ServerHubs to config!
[INFO @ 20:55:11 | Beat Saber Multiplayer] Writing to File @ H:\Steam\steamapps\common\Beat Saber\UserData\BeatSaberMultiplayer.json
[INFO @ 20:55:11 | Beat Saber Multiplayer] Found 0 presets in RoomPresets folder
[INFO @ 20:55:11 | Beat Saber Multiplayer] Downloading scrapped data...
[INFO @ 20:55:14 | Beat Saber Multiplayer] Received response from github.com...
[INFO @ 20:55:14 | Beat Saber Multiplayer] Parsing scrapped data...
[INFO @ 20:55:14 | Beat Saber Multiplayer] Scrapped data parsed! Time: 0.58s
[INFO @ 20:55:32 | Beat Saber Multiplayer] Hashing all avatars...
[INFO @ 20:55:32 | Beat Saber Multiplayer] All avatars hashed!
[INFO @ 20:55:32 | Beat Saber Multiplayer] Checking for updates...
[INFO @ 20:55:32 | Beat Saber Multiplayer] Loading all avatars...
[INFO @ 20:55:32 | Beat Saber Multiplayer] Loading text shader asset bundle...
[INFO @ 20:55:33 | Beat Saber Multiplayer] Loaded avatar "No Avatar"!
[INFO @ 20:55:33 | Beat Saber Multiplayer] Loaded avatar "Template"!
[INFO @ 20:55:33 | Beat Saber Multiplayer] Loaded avatar "Ooka Miko"!
[INFO @ 20:55:41 | Beat Saber Multiplayer] Requesting channels list...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Requesting channel infos...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Creating message...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Connecting to hub.assistant.moe:3700...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Creating message...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Connecting to hub.auros.red:3700...
[INFO @ 20:55:42 | Beat Saber Multiplayer] Requested channel infos!
[WARNING @ 20:55:42 | Beat Saber Multiplayer] Channel exception: System.Exception: Channel refused connection!
[WARNING @ 20:55:48 | Beat Saber Multiplayer] Channel exception: System.Exception: Channel refused connection!
```
